identity/client/src/pages/authorizations/components.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+import { TabListVertical, TabPanel } from "@carbon/react";
+import { label02, textSecondary } from "@carbon/elements";
+
+export const TabsTitle = styled.p`
+  position: relative;
+  top: 30px;
+  font-size: ${label02.fontSize};
+  font-weight: ${label02.fontWeight};
+  letter-spacing: ${label02.letterSpacing};
+  line-height: ${label02.lineHeight};
+  color: ${textSecondary};
+`;
+
+export const TabsContainer = styled.div`
+  .cds--css-grid {
+    margin-inline: 0;
+    padding-inline: 0;
+    max-inline-size: none;
+  }
+`;
+
+export const CustomTabListVertical = styled(TabListVertical)`
+  // Tab selectors container
+  &.cds--tabs--vertical {
+    margin-top: 50px;
+    background: var(--cds-layer-02);
+    border: none;
+    box-shadow: none;
+    padding-right: 24px;
+
+    // Tab selector
+    .cds--tabs__nav-item {
+      background: var(--cds-layer-02);
+      border: none;
+      margin-right: 24px;
+      height: 32px;
```

Comment:
For consistency, it might make sense to use CDS variables instead of these pixel values here as well (I'm aware I was the one who put them in pixels while developing the v1, but the intention was to eventually use the design system variables 😄).